app/src/test/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItemTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.simple.clinic.bp.history.adapter
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.util.TestUtcClock
+import org.threeten.bp.Duration
+import org.threeten.bp.Instant
+import org.threeten.bp.temporal.ChronoUnit.MINUTES
+import java.util.UUID
+
+class BloodPressureHistoryListItemTest {
+
+  @Test
+  fun `if blood pressure is editable, then user can edit blood pressure`() {
+    // when
+    val createdAt = Instant.parse("2020-01-01T00:00:00Z")
+    val bloodPressureNow = PatientMocker.bp(
+        uuid = UUID.fromString("78e876ae-3055-43d6-a132-7ad5dd930e23"),
+        systolic = 120,
+        diastolic = 70,
+        createdAt = createdAt
+    )
+    val bloodPressureADayInFuture = PatientMocker.bp(
```

Comment:
The name does not match the time that was used to create the BP (`aDayInTheFuture` vs `15 minutes`).